config/test.exs

```diff
@@ -1,12 +1,10 @@
 use Mix.Config
 
-config :fusion_auth, :tesla, adapter: Tesla.Mock
-
 config :logger, level: :error
 
 config :fusion_auth,
+  test_url: System.get_env("FUSION_AUTH_TEST_URL"),
```

Comment:
We should be fine just overriding the same `api_url` config with a `.env.test` file, no?